README.md

```diff
@@ -39,6 +39,13 @@ This results in a `julius` executable.
 
 To build the Vita or Switch version, use `cmake .. -DVITA_BUILD=ON` or `cmake .. -DSWITCH_BUILD=ON` instead of `cmake ..`.
 
+For more detailed instructions on how to build Julius, please check the following documents:
+- [Building for Windows](doc/building_windows.md);
```

Comment:
Trailing `;` for every item looks redundant: just end the bullet item without any punctuation.

Reply:
Will do! (This answer applies to every single one of your "trailing `;`" comments)